docs/2.0/docs/pipelines/installation/addingnewgitlabrepo.md

```
@@ -0,0 +1,547 @@
+################################################################################
+
+# The AWS partition to use.
+# AWSPartition: aws
+```
+
+#### Generate the repository contents
+
+1. Run the following command, from the root of your project, to generate the `infrastructure-live-access-control` repository contents:
+
+    <!-- TODO: Update template version before merging -->
+    ```bash
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)